eng/Version-Increment.ps1

```diff
@@ -0,0 +1,59 @@
+<#
+.SYNOPSIS
+Bumps up package versions after release
+
+.DESCRIPTION
+This script bumps up package versions following conventions defined at https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release-net
+
+.PARAMETER PackageCsprojPath
+Link to the package csproj
+
+.EXAMPLE
+Updating package version for core
+
+Version-Increment.ps1 -PackageCsprojPath "C:\Git\azure-sdk-for-net\sdk\core\Azure.Core\src\Azure.Core.csproj"
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Position=0)]
+    [ValidateNotNullOrEmpty()]
+    [string] $PackageCsprojPath
```

Comment:
I think we are going to want to do this based on package name and we can try and find the matching project in this script. Otherwise all the callers will have to try and do that in order to call this script.